service/collector_windows.go

```diff
@@ -79,36 +79,36 @@ func (s *WindowsService) Execute(args []string, requests <-chan svc.ChangeReques
 	return false, 0
 }
 
-func (s *WindowsService) start(elog *eventlog.Log, appErrorChannel chan error) error {
+func (s *WindowsService) start(elog *eventlog.Log, colErrorChannel chan error) error {
```

Comment:
We probably want to rename the WindowsService to WindowsCollector.

Reply:
Maybe, ignore this for now and we can take another loop if any naming is required before we cut the stable release.